tools/xtb/xtb_molecular_optimization.xml

```diff
@@ -0,0 +1,73 @@
+<tool id="xtb_molecular_optimization" name="xtb molecular optimization" version="0.1.0+galaxy0" profile="21.05">
```

Comment:
should the version here maybe corresponds to the xtb version? If so can you maybe use a macro?

Reply:
Yeah, I agree - the version needs to be updated and it should be moved to a macro.

Reply:
Macros created [7638ea6](https://github.com/RECETOX/galaxytools/pull/394/commits/7638ea623cbc28d6e4a2787a408b95fec22dc8a4) & [a9a6e65](https://github.com/RECETOX/galaxytools/pull/394/commits/a9a6e65c379ff56865bb1ed48f7b524f86e1d28a)